blog-service/2017/12-31.md

```diff
@@ -254,7 +254,7 @@ Bug Fix - In-product notification icons now display correctly.
 ---
 ## October 17, 2017
 
-**Dashboard sharing enhancements.** You can now share a dashboard with static timeranges provided in the URL itself, allowing you to share a snapshot in time of a dashboard with other users. For details, see [sharing or embedding](/docs/dashboards/share-dashboard-inside-org) a dashboard.
+**Dashboard sharing enhancements.** You can now share a dashboard with static timeranges provided in the URL itself, allowing you to share a snapshot in time of a dashboard with other users. For details, see [sharing or embedding](/docs/dashboards-classic/share-dashboard-inside-org) a dashboard.
```

Comment:
@JV0812 did you move the files so these updated links still point to a valid page?